Automattic/wp-calypso
# Site Profiler: Add a new component for landing page header

Resolve https://github.com/Automattic/dotcom-forge/issues/7296

## Proposed Changes

* Create a new header for the landing page based on the existing `domain-analyzer` component, [link here](https://github.com/Automattic/wp-calypso/blob/b359eb43a7aa2ed3d14381f683965e7f563eef05/client/site-profiler/components/domain-analyzer)

|Desktop | Mobile|
|-------|------|
|   ![CleanShot 2024-05-21 at 11 40 59@2x](https://github.com/Automattic/wp-calypso/assets/3519124/0183db03-38c7-4be5-b7a5-513d0113f374)|      ![CleanShot 2024-05-21 at 11 41 21@2x](https://github.com/Automattic/wp-calypso/assets/3519124/0982af58-6cd5-43c6-b6a3-79f0a3fa4fa7)        |

## Why are these changes being made?


* To implement the new designs

## Testing Instructions



* Apply this patch and navigate in your local to `/site-profiler`
* Make sure the design of the landing page header, which implies the title, paragraph under the title, input form and button correspond to the design in both Mobile and Desktop
* Note: The margins and padding have been kept similar to the previous component. They have not been matched to the pixel in the design as they look fine to me, but happy to change them if required. The main changes are in colors, copy, and typography. 

## Pre-merge Checklist



~~- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?~~
~~- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?~~
